src/styles/main.scss

```diff
@@ -151,17 +151,57 @@ $background-color: #f9f9f9;
       }
     }
 
-    > .slider-button {
-      background-color: white;
+    > .slider-button, > .help-container>.help-button {
```

Comment:
Nit: Space between operators and maybe new line so this is easier to read
```
> .slider-button, 
> .help-container > .help-button {
```
(Is it specific enough to just use `.slider-button, .help-button {`?)

Reply:
Thanks. Changed it to just `.slider-button, .help-button`. I've never been a fan of the `>` arrows anyway unless they're absolutely necessary.